### DISCUSSION TITLE
Much larger filesize compared to the command line

### DISCUSSION THREAD

**Discussion by peterdongan** [dlemstra/Magick.NET#928] [Help] [answered]
920px.jpg is 281k

command line:

`magick convert 1920px.jpg -resize 720x720 -quality 80 im2.jpg`

results in a 40k file, which is ideal for my purposes.

Magick.Net with the same size and quality values:
```
    using (var image = new MagickImage(sImage))
    {
        var size = new MagickGeometry(maxWidth, maxHeight);
        size.Less = false;
        image.Quality = quality;
        image.Resize(size);

        return image.ToByteArray();
    }
```

results in a 224k file, which is too big for my purposes.

I would have thought the result should be the same? Am I doing something wrong?

**Comment by dlemstra** [comment]
Can you share your input image? And why are you setting `size.Less`?

**Reply by peterdongan** [reply]
![1920px](https://user-images.githubusercontent.com/32374349/118357581-c02ce900-b572-11eb-9320-c8852783e4ad.jpg)

I'm setting `size.Less` because I only want to shrink images, not enlarge them.

**Comment by dlemstra** [comment, marked as answer]
The size of `image.ToByteArray().Length` is `40211` when I test this with the latest version of Magick.NET. Don't know why you are getting a different result. And you will need to `size.Less` to `true` if you want to enable that option.

**Reply by peterdongan** [reply]
Thank you for your help and quick responses. I see now that I had tested it with the .png version of the same image by mistake.
